src/transactions/feed/TransactionFeedV2.test.tsx

```diff
@@ -597,7 +575,7 @@ describe('TransactionFeedV2', () => {
       crossChainFeeAmount: '0.5',
       crossChainFeeAmountUsd: 500,
     })
-    expect(AppAnalytics.track).toBeCalledTimes(1)
+    expect(AppAnalytics.track).toBeCalledTimes(2)
```

Comment:
is this change expected? if so, can we assert or at least document why? Since we only assert on the swap event

Reply:
yes, a get started impression analytics event is firing since the get started component shows at the start of the test. Added an assertion for it.